src/main/java/edu/kis/powp/jobs2d/command/ComplexCommand.java

```diff
@@ -0,0 +1,16 @@
+package edu.kis.powp.jobs2d.command;
+
+import java.util.ArrayList;
+
+public class ComplexCommand implements DriverCommand {
+
+    ArrayList<DriverCommand> commands;
```

Comment:
should be private
List type (attribute and in the constructor) would be sufficient and would not limit the implementation.